android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java

```diff
@@ -81,8 +82,6 @@
     Images.Media.DATE_ADDED,
     MediaStore.MediaColumns.WIDTH,
     MediaStore.MediaColumns.HEIGHT,
-    Images.Media.LONGITUDE,
```

Comment:
These are not used anywhere?

Reply:
These columns are requested in the query but the value is never read anywhere since these values are retrieved in a different way: https://github.com/relivecc/react-native-cameraroll/pull/2/files/dd1205d6c01835009657d0d71bb1c5647c3a7331#diff-755efe7d85cd2c890eb71bbd89cc8abeR613